README.rst

```diff
@@ -78,22 +76,41 @@ All done! The VM contains all the required dependencies pre-installed.
 Manual
 ^^^^^^
 
+Instructions here have been tested on Fedora 30, using Python3
+
+Get git and python3:
+
+    sudo dnf install git python3
+
 Clone the source::
 
- git clone https://github.com/fedora-infra/nuancier.git
+    git clone https://github.com/fedora-infra/nuancier.git
 
+Install Python requirements
 
+    pip3 install -r requirements.txt
+ 
 Create the database scheme::
 
- python createdb.py
-
+    python3 createdb.py
 
-Run the server::
+Run the server locally::
 
- python runserver.py
+    python3 runserver.py
 
 You should be able to access the server at http://localhost:5000
 
+or to run the server externally, first open relevant port in firewall::
+
+    sudo firewall-cmd --add-port 5000/tcp --permanent
+    sudo firewall-cmd --reload 
```

Comment:
:-1: let's not advice people to poke holes into their firewall, ``runserver.py`` has a ``--port``argument iirc and if there is a port conflict with apache, when nuancier should be run via wsgi or apache configured to do port redirection, but I don't think this belongs to the documentation on how to get nuancier running for **development**.

Reply:
Thanks.